packages/nimble-tokens/CONTRIBUTING.md

```diff
@@ -5,18 +5,16 @@
 | Folder               | Description                             |
 | -------------------- | --------------------------------------- |
 | build                | Build scripts for generating files      |
-| data                 | _Managed by Adobe XD DSP plugin_        |
 | dist/fonts           | Fonts for use in applications           |
 | dist/icons           | Icons for use in applications           |
-| dist/styledictionary | _Managed by Adobe XD DSP plugin_        |
+| dist/styledictionary | JSON token source files                 |
 | docs                 | Files used by the documentation         |
 | NimbleTokens         | Project for building the Nuget package  |
 | source/icons         | Illustrator files for editing icons     |
 
 ## Getting started
 
 1. Build the monorepo, see [Getting Started](/CONTRIBUTING.md#getting-started)
-2. Ensure you have the [Adobe XD extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=Adobe.xd&ssr=false#overview) installed.
```

Comment:
Should we toss `packages\nimble-components\docs\xd-tips.md` and the reference to it in this file or is it still helpful to keep around since we have not fully migrated off xd yet?

Reply:
I'd vote to remove it. Even though we still have some links to XD I don't think devs are referring to those designs often / ever.

Reply:
Removed - 👍